src/shared_utilities/boundary_conditions.jl

```diff
@@ -84,3 +87,69 @@ function boundary_flux(
     Δz,
     _...,
 )::ClimaCore.Fields.Field end
+
+
+"""
+    boundary_vars(::AbstractBC , ::ClimaLand.TopBoundary)
+
+The list of symbols for additional variables to add to the
+model auxiliary state, which defaults to adding storage for the
+ top boundary flux fields,
+but which can be extended depending on the type of boundary condition used.
+
+Note that `:top_bc` must be present, with the default type and domain name,
```

Comment:
I'll note that this is not required for all models and currently used by soil & soil co2